ArcGISRuntimeSDKQt_QMLSamples/Maps/ShowLocationHistory/ShowLocationHistory.qml

```diff
@@ -0,0 +1,115 @@
+        }
+
+        GraphicsOverlay {
+            id: locationHistoryLineOverlay
+            SimpleRenderer {
+                SimpleLineSymbol {
+                    color: "lime"
+                    style: Enums.SimpleLineSymbolStyleSolid
+                    width: 2
+                }
+            }
+        }
+
+        GraphicsOverlay {
+            id: locationHistoryOverlay
+            SimpleRenderer {
+                SimpleMarkerSymbol {
+                    color: "red"
+                    style: Enums.SimpleMarkerSymbolStyleCircle
+                    size: 3
+                }
+            }
+        }
+    }
+
+    PolylineBuilder {
+        id: polylineBuilder
+        spatialReference: SpatialReference.createWebMercator()
```

Comment:
we shouldn't use these create methods for a spatial reference in a declarative block like this - better to declare a SpatialReference item with a wkid